ZenPacks/zenoss/LinuxMonitor/modeler/plugins/zenoss/cmd/linux/os_service.py

```diff
@@ -143,7 +143,7 @@
                         'Loaded:\s(?P<loaded_status>\w.+\))\s+'          # loaded status
                         '(Drop-In:(?P<drop_in>\s/\w[\w./].+)\s+)?'       # optional Drop-In
                         'Active:\s+'
-                        '(?P<active_status>\w+\s\(\w+\)(\ssince.+ago)?)' # active status
+                        '(?P<active_status>\w+\s\(\w+(:\s\w+-\w+)?\)(\ssince.+ago)?)' # active status
```

Comment:
This regex is overly compex if you are simply trying to grab that line after "Active:\s+".
If not, it needs some more explanation so that others can understand it.

Reply:
Prior to this fix, only strings like this were matched:
`Active: active (running) since Tue 2017-09-05 14:52:45 CDT; 6 months 1 days ago`
But not this:
`Active: failed (Result: exit-code) since Tue 2018-03-06 12:03:59 CST; 2 `days` ago`
The string it's not specifically matching is `(Result: exit-code)`

The new regex, (:\s\w+-\w+)? will match <: exit-code> if it exists. When this matches, I can grab the active status of a service that's failed which I could not before.

Reply:
If that is the case, you should break up the regex and grab what you need explicitly.

In any case, you need to explain this inside the code so that the next developer can clearly understand what is going on.

Reply:
Made sure to put in all examples of the strings this can match agaist in the docstring.